README.md

```diff
@@ -33,8 +34,8 @@ from kfcicli.main import KubeflowCI
 from pathlib import Path
 
 client = KubeflowCI(
-    modules=[Path(f"/../charmed-kubeflow-solutions/modules/kubeflow/applications.tf")],
-    base_path=Path({tmp_folder}),
+    modules=[Path(f"../charmed-kubeflow-solutions/modules/kubeflow/applications.tf")],
+    base_path=Path(f"tmp_folder"),
```

Comment:
```suggestion
    base_path=Path(f"{tmp_folder}"),
```

Reply:
Good catch, went with using $HOME

Reply:
can you add this? I believe it is needed